src/bitmessagekivy/baseclass/myaddress_widgets.py

```diff
@@ -0,0 +1,64 @@
+# pylint: disable=too-many-arguments, no-name-in-module, import-error
+# pylint: disable=too-few-public-methods, no-member, too-many-ancestors
+
+"""
+MyAddress widgets are here.
+"""
+
+from kivymd.uix.button import MDFlatButton
+from kivymd.uix.dialog import MDDialog
+from kivymd.uix.label import MDLabel
+from kivymd.uix.list import IRightBodyTouch
+
+from pybitmessage.bitmessagekivy.get_platform import platform
+from pybitmessage.bitmessagekivy.baseclass.common import ThemeClsColor
+
+
+class BadgeText(IRightBodyTouch, MDLabel):
+    """BadgeText class for kivy UI"""
+
+
+class HelperMyAddress(object):
+    """Widget used in MyAddress are here"""
+    dialog_height = .25
+
+    @staticmethod
+    def is_active_badge():
+        """This function show the 'active' label of active Address."""
+        active_status = 'Active'
+        is_android_width = 90
+        other_platform_width = 50
+        other_platform_height = 60
```

Comment:
should be just `height` as it doesn't depend on platform

Reply:
Done